src/nvim/api/autocmd.c

```diff
@@ -322,23 +322,6 @@ Array nvim_get_autocmds(Dict(get_autocmds) *opts, Arena *arena, Error *err)
         PUT_C(autocmd_info, "buflocal", BOOLEAN_OBJ(false));
       }
 
-      // TODO(sctx): It would be good to unify script_ctx to actually work with lua
-      //  right now it's just super weird, and never really gives you the info that
-      //  you would expect from this.
```

Comment:
This comment is out of date, probably before the `nvim -V1` handling. And it's too verbose anyway.